fuzz/fuzz_targets/rabbit.rs

```diff
@@ -0,0 +1,60 @@
+use honggfuzz::fuzz;
```

Comment:
Asking again for confirmation of fixed random numbers dependent on test case/seed.